jtabba/CDW-1
# CDW 38 - Create UI page that lists mentors

## Type of change

[x] New feature
<br />
[] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure


## Description

Implements a mentor page (with dummy data) that lists all mentors in our database. Also implements a very basic sort based on the AreasOfExpertise attribute on our Mentor model.

## Ticket link

[CDW 38 - Create UI page that lists mentors](https://linear.app/cdw/team/CDW/active )

## Part of a back end implementation?
[x] No
<br />
[] [Yes]()  


## How to test


1. Checkout to this branch, CD into /frontend, ```npm start```
2. Navigate to /mentors
3. Check that 8 mentors are showing up
4. Click between categories in filter to see how the mentors are filtered out based on area of expertise.

## Screenshots

![image](https://github.com/jtabba/CDW-1/assets/12403004/1915e52f-4a3b-4f3e-b38d-41f770657019)
![image](https://github.com/jtabba/CDW-1/assets/12403004/9f6bd5d0-c7fe-4a0a-927c-568e2cb3a16a)
![image](https://github.com/jtabba/CDW-1/assets/12403004/b3e8a3ea-000e-4cf2-ae54-ecfe149b9329)
